.eslintrc.js

```diff
@@ -54,7 +54,6 @@ module.exports = {
     'no-underscore-dangle': 'error',
     'prefer-promise-reject-errors': 'error',
     'prefer-template': 'error',
-    'sort-imports': 'error',
```

Comment:
Please revert the changes in this file before committing, especially in favor of https://github.com/GoogleChromeLabs/chromium-bidi/pull/523 which already does this.